### PR TITLE
🌐 Add French definition for "gender pronouns"

### PR DESCRIPTION
**Pull request by CharlesMangwa**
Hello!

This PR aims to add the French definition for **"gender pronouns"**. 

A link has been added to the existing English definition (and vice versa).

#### Notes

To the editor who would have time to go through this PR, I wasn't too sure about some specific terms but here are the resources I based my translations on:

* "non-conformité de genre" for "gender non-conforming": https://fr.wikipedia.org/wiki/Variance_de_genre
* "agenre" for "agender": https://fr.wiktionary.org/wiki/agenre

Thanks in advance! ✨

### CODE REVIEW

File: 11ty/definitions/fr_FR/pronoms-de-genre.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Pronoms de genre
```
Comment (deammer) [inline 11ty/definitions/fr_FR/pronoms-de-genre.md]: On appelle plutot ca les "pronoms genrés" en francais, je crois 🤔 qu'en penses-tu?
Par exemple: http://www.slate.fr/story/181860/etats-unis-genre-anglais-pronom-non-binaire-they-dictionnaire-merriam-webster

**Comment by CharlesMangwa** [comment]
Just implemented some changes suggested by @deammer. 

(I was hesitating with the translation you proposed so I'd agree to go for that one instead!)

**Review by deammer — APPROVED** [review]

**Comment by tatianamac** [comment]
@CharlesMangwa Could we please add the two resources you cited in the description within the body of the text? Whereever we can provide additional resources in the language of the definition, I'd like to do so. Thank you!

**Comment by CharlesMangwa** [comment]
@tatianamac Just added my two resources and the one @deammer mentioned in his feedback, to the Front Matter `reading` section. I completely forgot that was a thing even though I saw it in other definition files. Will use it more often in the future!

Oh, and I also made the title lowercase here too!

**Review by tatianamac — APPROVED** [review]

**Pull request merged**